assets/queries/cloudFormation/sbd_domain_declared_as_a_resource/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "sbd_dmain_declared_as_a_resource",
+  "queryName": "SDB Domain Declared As A Resource",
+  "severity": "LOW",
+  "category": null,
+  "descriptionText": "SimpleDb Domain resource should not be declared",
+  "descriptionUrl": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-simpledb.html"
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline